app/javascript/tests/close_contact/CloseContactTable.test.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { expect } from '@jest/globals';
+import { shallow, mount } from 'enzyme';
+import { Card, InputGroup } from 'react-bootstrap';
+import _ from 'lodash';
+
+import InfoTooltip from '../../components/util/InfoTooltip';
+import CloseContactTable from '../../components/patient/close_contacts/CloseContactTable';
+import CloseContactModal from '../../components/patient/close_contacts/CloseContactModal';
+import { mockPatient1, mockPatient2 } from '../mocks/mockPatients';
+import { formatPhoneNumber } from '../../utils/Patient';
+import * as mockCloseContacts from '../mocks/mockCloseContact';
+
+const authyToken = 'Q1z4yZXLdN+tZod6dBSIlMbZ3yWAUFdY44U06QWffEP76nx1WGMHIz8rYxEUZsl9sspS3ePF2ZNmSue8wFpJGg==';
+const ASSIGNED_USERS = [123234, 512678, 910132];
+
+function getShallowWrapper(mockPatient, canEnroll, assigned_users) {
+  return shallow(<CloseContactTable patient={mockPatient} authenticity_token={authyToken} can_enroll_close_contacts={canEnroll} assigned_users={assigned_users} />);
+}
+
+function getMountedWrapper(mockPatient, canEnroll, assigned_users) {
+  let wrapper = mount(<CloseContactTable patient={mockPatient} authenticity_token={authyToken} can_enroll_close_contacts={canEnroll} assigned_users={assigned_users} />);
+  // The Table Data is loaded asynchronously, so we have to mock it
+  const closeContactsOfPatient = _.values(mockCloseContacts).filter(x => x.id === mockPatient.id);
+  const tableData = wrapper.state('table');
+  tableData.rowData = closeContactsOfPatient;
+  wrapper.setState({ table: tableData });
+  return wrapper;
+}
+
+describe('CloseContactTable', () => {
+  it('Properly renders all main components for empty close contact', () => {
+    const wrapper = getShallowWrapper(mockPatient1, true, ASSIGNED_USERS);
+    expect(wrapper.find(Card).exists()).toBeTruthy();
+    expect(wrapper.find(Card.Header).exists()).toBeTruthy();
+    expect(wrapper.find(Card.Header).text()).toContain('Close Contacts');
+    expect(wrapper.find('CardHeader').containsMatchingElement(<InfoTooltip />)).toBeTruthy();
+    expect(wrapper.find('Button').at(0).text()).toContain('Add New Close Contact');
+    expect(wrapper.find(InputGroup).exists()).toBeTruthy();
+    expect(wrapper.find(CloseContactModal).exists()).toBeFalsy();
+  });
+
+  it('Properly renders the Close Contact Modal when clicking the `Add New Close Contact` button', () => {
```

Comment:
I think you meant for this to be Opens the CC modal when...

I'd also add a separate test for the modal rendering

Reply:
Done, but not sure what you mean by the second bit. I have a whole test suite for the modal

Reply:
this was definitely my friday brain, my bad